docs/assembly-control-unit.md

```diff
@@ -171,47 +281,105 @@ Heat  |  Stir |
+  - Take the other end of the 4-wire cable and place the black and green wire in their own slots of the 4-to-1 wire connector. Then put the red and blue wires together in one slot. That leaves the last slot open for power (-12V), put the 4" black wire in that slot.
+
+  <IMG ALT="Wire the negative Phoenix wires and connector" SRC="./media/control-unit/neg12v.png" WIDTH="400" />
+
+
+  - Wiring the relay   
+
+    <IMG ALT="4-channel relay" SRC="./media/control-unit/relay.png" WIDTH="300" />
+
+     - Take the female phoneix connector and insert each of its 4" positive wires into the "NO" (Normally Open) terminals on the 4-channel relay board. The terminals are labeled "NO1", NO1 as so on. Screw the terminals down gently but firmly.
+        <IMG ALT="Wire the negative Phoenix wires and connector" SRC="./media/control-unit/pos12v.png" WIDTH="500" />
+     - Use the 3-wire cable, put each of its wires into the terminals labeled "COM" (common) and tighten the terminals.
+
+     - Take the other end of the 3-wire cable and insert each of its wires into a wire connector. In the remaining open slot, insert one of the 4" red wires, which will later connect to the 12v manifold.
+  <IMG ALT="Wire the negative Phoenix wires and connector" SRC="./media/control-unit/3wire.png" WIDTH="400" />
+
+
 - Wiring the 5v half of the sub-assembly
-  - Each relay board will have a (DC+), (DC-), and (IN) terminal. Insert and secure red wires into the (DC+), black wires into the (DC-), and the male end of the male-to-female breadboard headers into the (IN) terminal. A 4x relay board will have IN1, IN2, IN3, and IN4 - insert a male header into each: IN1, IN2, and IN3.
-    - We highly recommend gently bending the male header to a 90 degree angle before doing the previous step for reasons that will become clear in the following steps.
-  - To conserve space and prevent short circuits, we also recommend using flush cutters to remove excess material from the underside of the boards. Once this is complete, use electrical tape to cover over the bottom of the boards.
+  - We highly recommend gently bending the male headers of the jumper wires to a 90 degree angle with a needle nose pilers before inserting them in the terminals.
+
+    <IMG ALT="Wire the negative Phoenix wires and connector" SRC="./media/control-unit/relay_term.png" WIDTH="400" />
+  - The relay board will has:
+      - (DC+) terminal - insert a 4" red wire
+      - (DC-) terminal - insert a 4" black wire
+      - (IN1) terminal - insert the male end of the Jumper wire
+      - (IN2) terminal - insert the male end of the Jumper wire
+      - (IN3) terminal - insert the male end of the Jumper wire
+      - (IN4) terminal -  [no action]
+  - To conserve space and prevent short circuits, we recommend using flush cutters to remove excess material from the underside of the boards. Once this is complete, use electrical tape to cover over the bottom of the boards.
```

Comment:
Once this is done, the relay board should fit in the printable "relay board case" sub-component (link: https://github.com/FourThievesVinegar/microlab-parts/tree/master/v6/control-box/sub-components)

Reply:
With relay case do we still need to "Gently bend all wires on both sides of the relay up and wrap them with narrow (half width) strips of electrical tape. Tape prevents shorts and helps keep the wires from being damaged as you arrange components in the case."